chromium_src/third_party/rust/serde_json_lenient/v0_2/wrapper/large_integers.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// This source provides extra C++ bindings which are used to handle 64bit
+/// integers over base::Value types.
+#[cxx::bridge(namespace=serde_json_lenient)]
+mod ffi {
```

Comment:
@cdesouza-chromium you should be able to add `#[cfg(feature = "json_64bit_int_support)]` on this `ffi` module too, so it wouldn't have to be linked when the feature is disabled

Reply:
I wish I could, but unfortunately it confuses `cxxbridge` codegen entirely, and nothing comes out from it once you have the `[cfg(...)]` in there.